### PR TITLE
Use new angular-bootstrap attribute directive instead of element dire…

### PR DESCRIPTION
**Pull request by B3nCr**
…ctive for uib-timepicker in bs-time directive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,7 +38,7 @@
     <!--app-->
     <script src="docs/js/app.js"></script>
     <!--xeditable-->
-    <script src="dist/js/xeditable.min.js"></script>
+    <script src="dist/js/xeditable.js"></script>
```
Comment (B3nCr) [inline index.html]: Sorry, I forgot to change this back when I was debugging the model binding issue that's in the bstime directive. I can change it back, it'll add another commit and hopefully you can squash it when you merge the PR, or I can leave it like it is.

Comment (eugef) [inline index.html]: It is ok to include non-minified version

**Comment by eugef** [comment]
@B3nCr, @ckosloski what about tests? are they green?

**Comment by B3nCr** [comment]
Hi

Sorry, i mentioned this in the bug report.

I can't get the tests passing in the master branch so they're still the
same here.

I'm wondering if it's because my npm version is not putting files in the
same place because I had a problem with grunt build and jsdoc not being
found either.

Perhaps you could pull my branch and see if you can run them.

Thanks
Ben

**Comment by B3nCr** [comment]
This is my node and npm version for comparison.

```
D:\dev\Repos\angular-xeditable [squashed-bs-time]> node -v
v6.9.5
D:\dev\Repos\angular-xeditable [squashed-bs-time]> npm -v
4.0.5
```

**Comment by eugef** [comment]
@B3nCr npm v2 is required to run build and tests.

**Comment by B3nCr** [comment]
@eugef Yes, that's what I thought. 

Well I'll leave this PR for you to do whatever you choose. I can't see an easy way of installing NPM v2 and v4 side by side. It shouldn't be that difficult for someone to pull my branch and run the tests.

Sorry I couldn't be of more help.

**Comment by B3nCr** [comment]
Perhaps to make someone else running the tests easier I could create a PR into a different branch in the vitalets repo then it won't be necessary to clone my repo, just checkout a branch.

**Comment by B3nCr** [comment]
I'll see if I can get Travis CI to build my branch.

**Comment by eugef** [comment]
Maybe @ckosloski could run the tests agains this PR?

**Comment by ckosloski** [comment]
All tests pass.
I also manually tested all bootstrap directives and those passed as well.
This will close #559

**Comment by eugef** [comment]
Great, then I merge it.

**Comment by B3nCr** [comment]
Thanks. 

I know this is merged but is this an appropriate place to discuss perhaps getting a travis-ci build setup that runs the tests? Is that something that anyone would be interested in?

I've set one up that runs `grunt build` and it seems to work fine but I'm not sure how to run the tests from the command line.

https://travis-ci.org/B3nCr/angular-xeditable

**Comment by eugef** [comment]
As far i know tests can be run only in browser. Can Travis-CI do it?

**Comment by B3nCr** [comment]
I'm not sure yet, it seems like it can but I was hoping there would be a
command line way that I could experiment with first.

The tests seem to be using the old angular-scenario e2e testing.

I could take a look at updating the tests to use protractor at the same
time, there seems to be a lot more documentation for Travis+Protractor?

Another question, more of a side question. Of I was to start looking at the
protractor tests what's the best way to take my repo forwards? I've tried
to rebase the vitalets master back into my B3nCr master but now GitHub is
telling me they are diverged, the rebase must have created new commits (i
thought the point of rebase was that it didn't do that).

Should I just drop my fork and create a fresh fork for future contributions?

**Comment by ckosloski** [comment]
This is what I do to rebase.
```
git fetch upstream
git rebase upstream/master
```

That would be great if you have time to update the tests!!!

**Comment by B3nCr** [comment]
I take it you manually add the upstream origin which refers to the the
source repo of the fork, my local repo only has one remote which is my
github repo,

**Comment by ckosloski** [comment]
I don't remember what I did, but I used those commands after I viewed this [stackoverflow](http://stackoverflow.com/questions/7244321/how-to-update-a-github-forked-repository)

**Pull request merged**